src/net.cpp

```diff
@@ -1156,7 +1156,7 @@ void CConnman::AcceptConnection(const ListenSocket& hListenSocket) {
 
     // don't accept incoming connections until fully synced
     if(fMasternodeMode && !masternodeSync.IsSynced()) {
-        LogPrintf("AcceptConnection -- masternode is not synced yet, skipping inbound connection attempt\n");
+        LogPrint("masternode", "AcceptConnection -- masternode is not synced yet, skipping inbound connection attempt\n");
```

Comment:
Not sure this should be `masternode` category. When debugging connection issues, people will usually only enable the `net` category and this way they'll never see this message even if it's the issue they're looking for.